src/covid19sim/base.py

```diff
@@ -26,53 +26,53 @@ def __init__(self, initial_timestamp):
         Args:
             initial_timestamp (datetime.datetime): The environment's initial timestamp
         """
-        super().__init__()
-        self.initial_timestamp = initial_timestamp
-
-    def time(self):
-        """
-        Alias for simpy-defined `now` attribute
-
-        Returns:
-            datetime.datetime: current time in the environment
-        """
-        return self.now
+        self.initial_timestamp = datetime.datetime.combine(initial_timestamp.date(),
+                                                           datetime.time())
+        self.ts_initial = int(self.initial_timestamp.timestamp())
+        super().__init__(self.ts_initial)
 
     @property
     def timestamp(self):
         """
         Returns:
-            datetime.datetime: current date (initial timestamp shifted by
-                env.now * TICK_MINUTE minutes)
-        """
+            datetime.datetime: Current date.
+        """
+        #
+        ##
+        ## The following is preferable, but not equivalent to the initial
+        ## version, because timedelta ignores Daylight Saving Time.
+        ##
+        #
+        #return datetime.datetime.fromtimestamp(int(self.now))
+        #
```

Comment:
We don't really care about that! It's good!

Reply:
The use of `datetime.datetime.fromtimestamp()` means that sometimes the datetime appears to jump forward and backwards ("spring forward, fall back") with DST.

This means that in general, we must be careful about whether a `Human`'s actions are intended to be of a specified duration (such as shopping) or of a specified end-time (such as sleeping).

Right now no such distinction is being operated, and there's so much code using time deltas that I am hesitant to change this code, for fear of generating excess- or negative-length time-deltas.